integrations/http/http.go

```diff
@@ -35,9 +35,14 @@ var args = sdkmodule.WithArgs(
 	"url",
 	"params?",
 	"headers?",
-	"raw_body?",
```

Comment:
i think you might want to leave these in for backward compatibility unless @haimzlato doesn't have an issue with that

Reply:
discussed at sync - no backward compatibility. 
resolving.

Reply:
No need for backward compatibility, but must change all samples and example.